src/Modules/AuthModule/components/VerifyAccount/VerifyAccount.tsx

```diff
@@ -1,6 +1,81 @@
+import React, { useContext } from 'react';
+import { useForm } from 'react-hook-form';
+import img1 from "../../../../assets/images/PMS 3.png";
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { AuthContext } from '../../../../context/AuthContext';
 
 export default function VerifyAccount() {
-  return (
-    <div>VerifyAccount</div>
+  let {baseUrl}= useContext(AuthContext)
+
+    const { handleSubmit, register,formState: { errors } } = useForm();
+  const Navigate=  useNavigate()
+
+    async function onSubmit (data:any) {
+        try {
+            const res = await axios.put(`${baseUrl}/Users/verify`, data);
+            console.log(res);
```

Comment:
Logs should be removed from our branch before merging with the master to prevent revealing any sensitive data.